packages/fast-components-react-msft/src/carousel/carousel.tsx

```diff
@@ -223,6 +225,31 @@ class Carousel extends Foundation<
         return this.slides.length !== 1;
     }
 
+    /**
+     * Check if it is the FIRST slide for looping
+     */
+    private get isFirstSlide(): boolean {
+        if (this.props.items.length) {
```

Comment:
It seems like slides is always an array, unlike items which can be undefined, so I think you should change this to check the length on slides.